scripts/format.sh

```diff
@@ -17,7 +17,7 @@ else
 fi
 
 if [ "$1" != check ]; then
-    bazel build buildifier
+    bazel run buildifier
```

Comment:
It should be `run` not `build`. It never works.

Reply:
good catch